src/pykeen/training/training_loop.py

```diff
@@ -401,6 +411,15 @@ def _train(  # noqa: C901
         elif not self.model.supports_subbatching:
             raise SubBatchingNotSupportedError(self.model)
 
+        model_contains_batch_norm = any(
+            isinstance(m, (nn.BatchNorm1d, nn.BatchNorm2d, nn.BatchNorm3d, nn.SyncBatchNorm))
```

Comment:
haven't we written a function that checks this somewhere else in PyKEEN? maybe time to factor this one-liner out into a utility function

Reply:
we have a method which checks for compatibility with subbatching, which is happens to be a check for the same modules.

Reply:
Okay, better not conflate them. Keep
It apart

Reply:
4bcb78c